docs/3.x.x/installation-options/quick-start-detailed.md

```diff
@@ -0,0 +1,400 @@
+
+This page explains how to create a new project in Strapi. Each step from creating a new project to creating content types ready to be consumed is covered. ([Check out our video tutorial](https://www.youtube.com/watch?v=yMl5IcFHA74)).
+
+**Table of contents:**
+
+1. [Create a project](#_1-create-a-project)
+2. [Create an admin user](#_2-create-an-admin-user)
+3. [Create a Content Type](#_3-create-a-content-type)
+
+   - [Files structure](#files-structure)
+
+4. [Manage and add data to Content Type](#_4-manage-and-add-data-to-content-type)
+5. [Set roles and permissions](#_5-set-roles-and-permissions)
+6. [Consume the Content Type API](#_6-consume-the-content-type-api)
+
+   - [List entries (GET)](#list-entries-get)
+   - [Get a specific entry (GET)](#get-a-specific-entry-get)
+   - [Create data (POST)](#create-data-post)
+   - [Update data (PUT)](#update-data-put)
+   - [Delete data (DELETE)](#delete-data-delete)
+
+---
+
+## 1. Create a project
+
+Navigate to your parent "projects" directory in your command line.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$
+```
+
+Type the following in your command line and press enter to create a Strapi Quick Start project.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ strapi new my-project --quickstart
+```
+
+::: warning NOTE
+You can replace the "my-project" project name with any name you want.
+
+:::
+
+::: warning NOTE
+
+When you create a new Quick Start project in Strapi, you will download all the node modules, the Strapi files necessary and the Strapi plugin files. THIS STEP CAN TAKE SEVERAL MINUTES DEPENDING ON YOUR INTERNET CONNECTION SPEED. Please wait for the process to complete before cancelling or trying to continue.
+
+:::
+
+::: warning NOTE
+You can replace the "my-project" project name with any name you want.
+
+Quick start will install Strapi using a SQLite database. ([SQLite](https://www.sqlite.org/index.html) is likely used more than all other database engines combined.)
+
+:::
+
+You will see something like this:
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ strapi new my-project --quickstart
+🚀 Start creating your Strapi application. It might take a minute, please take a coffee ☕️
+
+
+⏳ Testing database connection...
+The app has been connected to the database successfully!
+
+🏗  Application generation:
+✔ Copy dashboard
+✔ Install plugin settings-manager.
+✔ Install plugin content-type-builder.
+✔ Install plugin content-manager.
+✔ Install plugin users-permissions.
+✔ Install plugin email.
+✔ Install plugin upload.
+✔ Link strapi dependency to the project.
+
+👌 Your new application my-project is ready at /home/str/Desktop/Strapi/Projects/my-project.
+
+⚡️ Change directory:
+$ cd my-project
+
+⚡️ Start application:
+$ strapi start
+```
+
+The command will automatically create a Strapi app folder within your parent "projects" directory.
+
+```bash
+str@pi:~/Desktop/Strapi/Projects$ ls
+my-project
+```
```

Comment:
```bash
ls
my-project
```

Reply:
This was just removed and reworked a different way.